src/Components/App.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import Areas from '../Components/Areas';
+import  getHomeRepairs  from '../apiCalls';
+import NavBar from './NavBar';
+import Videos from '../Videos';
+import Projects from './Projects';
+import { Route, Switch } from 'react-router-dom'
+import '../App.css';
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      areas: ['kitchen', 'bathroom', 'bedroom', 'miscellaneous'],
+      homeRepairs: []
+    }
+  }
+
+  componentDidMount() {
+    getHomeRepairs()
+    .then(data => this.setState({homeRepairs: data}))
+  }
+
+  returnProjects = (projects, category) => {
+     if(projects.length > 0) {
+      let filteredProjects = projects.filter(project => {
+          let lowerCaseArea = project.areaOfHome.toLowerCase()
+          let lowerCaseCategory = category.toLowerCase()
+            if(lowerCaseCategory === lowerCaseArea) {
+               return project;
+            }
+          })
+       return (
+       <Projects 
+       category={filteredProjects} />
+       )
+    }
+    }
+
+    findVideo = (repairs, project) => {
+      if(repairs.length > 0) {
+        let foundVideo = repairs.find(repair => {
+          if(project === repair.project) {
+            return repair
+          }
```

Comment:
Like the filter method above, I think you can remove the if block and just return the result of the condition (i.e `return project === repair.project`) and you should get the same result.